.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+**Test Configuration**:
+* Database and Version:
+* Alter-Wrapper?:
+* OS:
```

Comment:
```suggestion
**Test Configuration:**

- Database and Version:
- Alter-Wrapper?:
- OS:
```

MD032/blanks-around-lists: Lists should be surrounded by blank linesmarkdownlint[MD032](https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md032.md)

MD004/ul-style: Unordered list style [Expected: dash; Actual: asterisk]markdownlint[MD004](https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md004.md)

And  colon positioned `**Test Configuration**:` to `**Test Configuration:**`